scripts/update/Updater.php

```diff
@@ -57,9 +57,9 @@ public function update($initialVersion)
             $this->setVersion('2.6.1');
         }
 
-        $this->skip('2.6.1', '4.3.0');
```

Comment:
because of this tag https://github.com/oat-sa/extension-tao-outcomeui/releases/tag/v4.3.1.
you think we should release another hotfix instead 4.3.2 ?